modules/ROOT/pages/customer-connector-upgrade.adoc

```diff
@@ -0,0 +1,162 @@
+= Customer Guide for Upgrading and Testing Custom Connectors for Java 17
+
+Upgrade and test your custom connectors for Java 17 to ensure compatibility within the MuleSoft ecosystem. 
+
+== Before You Begin
+
+Before upgrading and testing your custom connector for Java 17, you must be familiar with the following:
+
+* *How to run and compile your custom connector*
++
+Ensure your custom connector runs with Java 17, but always use Java 8 as the target compilation level.
++
+Because you must compile your custom connector with Java 8, use the Java compiler configuration from the Java SDK parent POM. You can’t use language features from Java 11 or Java 17. 
+
+* *-add-opens or -add-export directives*
++
+Do not set `-add-opens` or `-add-export` directives.
+
+* *Minimum Mule version caveats*
```

Comment:
The other bullets also seem like caveats. I wonder if the intro should mention "considerations" and then remove this kind of language from the bullets?